plugins/artnet/ArtNetNode.cpp

```diff
@@ -1334,11 +1334,13 @@ void ArtNetNodeImpl::RDMRequestCompletion(
       // hopefully update the remote controller
       port->on_discover->Run();
     } else {
-      OLA_WARN << "ArtNet RDM request failed with code " << reply->StatusCode();
+      OLA_WARN << "Art-Net RDM request failed with code "
+               << reply->StatusCode();
```

Comment:
You've somehow inserted a Unicode character in here according to the compiler errors.

Reply:
Yes, it was an non-breaking space instead of a regular space.